src/test/regress/bin/normalize.sed

```diff
@@ -100,6 +100,14 @@ s/of relation ".*" violates not-null constraint/violates not-null constraint/g
 s/partition ".*" would be violated by some row/partition would be violated by some row/g
 s/of relation ".*" contains null values/contains null values/g
 
+# libpq message changes
+# We ignore multiline error messages, and substitute first line with a single line
+# alternative that is used in some older libpq versions.
```

Comment:
Lets make a note to check the need for these normalization rules when we drop pg13 support.

I believe the outputs we ought to be normalizing here changed between pg13 and pg14

Reply:
That's correct. The only failures we observed before the normalization rules were in PG13 schedules. I guess I should add this in the comments before the normalization rules.

Reply:
Using `PG_VERSION_13` etc. in the comments is useful for searching in the future

Reply:
In my opinion `pg13` is a better choice here as I expect the developer that want to drop support for `pg13` to review all occurrences of this in the codebase.

Reply:
I'd add both